tests/test_models.py

```diff
@@ -276,3 +285,23 @@ def test_document_get_text_window(test_document):
     text_window = test_document.get_text_window(text_block, (-2, 2))
     assert isinstance(text_window, str)
     assert len(text_window) > len(text_block.to_string())
+
+
+def test_to_huggingface(test_dataset, test_dataset_gst):
+    """Test that the HuggingFace dataset can be created."""
+    dataset_hf = test_dataset.to_huggingface()
+    dataset_gst_hf = test_dataset_gst.to_huggingface()
+    assert isinstance(dataset_hf, HuggingFaceDataset)
+    assert isinstance(dataset_gst_hf, HuggingFaceDataset)
+    assert len(dataset_hf) == sum(len(doc.text_blocks) for doc in test_dataset.documents if doc.text_blocks)
+    assert len(dataset_gst_hf) == sum(len(doc.text_blocks) for doc in test_dataset_gst.documents if doc.text_blocks)
+    assert list(dataset_hf.features.keys()) == ['text_block_id', 'text', 'type', 'language', 'type_confidence',
+                                                'page_number', 'coords', 'document_id', 'document_name',
+                                                'document_source_url', 'document_content_type', 'document_md5_sum',
+                                                'languages', 'translated', 'has_valid_text', 'publication_ts',
+                                                'geography', 'category', 'source', 'sectors', 'document_url',
+                                                'document_cdn_object', 'document_description', 'document_slug']
+    assert list(dataset_gst_hf.features.keys()) == ['text_block_id', 'text', 'type', 'language', 'type_confidence',
+                                                    'page_number', 'coords', 'document_id', 'document_name',
+                                                    'document_source_url', 'document_content_type', 'document_md5_sum',
+                                                    'languages', 'translated', 'has_valid_text', 'geography']
```

Comment:
you might be able to check against `model.__fields__` here to make this more dynamic against future pydantic model changes

Reply:
as in just checking the keys? think the other assert statements are still valuable